prow/npj/npj.go

```diff
@@ -115,3 +116,27 @@ func BatchSpec(p config.Presubmit, refs kube.Refs) kube.ProwJobSpec {
 	}
 	return pjs
 }
+
+// EnvironmentForSpec returns a mapping of environment variables
```

Comment:
Go style prefers short names to long. See the env names in the os package, such as [os.ExpandEnv](https://golang.org/pkg/os/#ExpandEnv).